Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -209,6 +209,8 @@ public override void OnPostSceneQuery()
         {
             base.OnPostSceneQuery();
 
+            BaseCursor?.SetVisibility(IsInteractionEnabled);
```

Comment:
I'll add a comment. Without this, when the poke pointer is disabled, the finger cursor still shows up. This ensures that we actually hide the finger cursor if we cannot poke things. Otherwise the system is being misleadding.